lib/media/streaming_engine.js

```diff
@@ -1368,6 +1375,34 @@ shaka.media.StreamingEngine = class {
     }
   }
 
+  /**
+   * @param {!BufferSource} rawResult
+   * @param {!shaka.media.SegmentReference} reference
+   * @param {!shaka.media.SegmentIterator} iter
+   * @return {!Promise.<!BufferSource>} finalResult
+   * @private
+   */
+  async aes128Decrypt_(rawResult, reference, iter) {
```

Comment:
It would be nice to have tests that covered this somehow.  I know StreamingEngine tests are a mess, but could you give it a shot?  If it can't be done in a day, then please file an issue under "type: code health" to follow up later.